client/src/app/management/components/committee-edit/committee-edit.component.ts

```diff
@@ -93,6 +94,34 @@ export class CommitteeEditComponent extends BaseModelContextComponent implements
         );
     }
 
+    /**
+     * Returns a function to know if the option which contains this committee should be disabled
+     * in the `receive_forwardings_from_committee_ids`-control
+     *
+     * @returns A function that will return a boolean
+     */
+    public getDisableOptionWhenFn(): (value: ViewCommittee) => boolean {
+        return value => {
+            return value.id === this.committeeId;
+        };
+    }
+
+    /**
+     * Returns a function to get a tooltip if the option which contains this committee is rendered
+     * in the `receive_forwardings_from_committee_ids`-control
+     *
+     * @returns A function that will return a tooltip
+     */
+    public getTooltipFn(): (value: ViewCommittee) => string | null {
+        return value => {
+            if (value.id === this.committeeId) {
+                return this.translate.instant('You can change this option only in the forwarding section.');
```

Comment:
The translation should happen using the translate pipe in HTML.
To mark something translatable you need to use the marker function.

```ts
import { marker as _ } from '@biesbjerg/ngx-translate-extract-marker';
```
and

```ts
return _('You can change this option only in the forwarding section.');
```